src/platform-includes/performance/opentelemetry-setup/with-java-agent/without-auto-init/java.spring-boot.mdx

```diff
@@ -0,0 +1,31 @@
+If you are not using the auto initialization provided by `sentry-opentelemetry-agent` (`SENTRY_AUTO_INIT=false`) you have to
+set the `instrumenter` option to `otel`. This disables all Sentry instrumentation and relies on the chosen OpenTelemetry tracers for creating spans.
+
+```properties {filename:application.properties}
+sentry.dsn=___DSN___
+sentry.traces-sample-rate=1.0
+# enable this to see more logs
+sentry.debug=false
+# set the instrumenter to use OpenTelemetry instead of Sentry
+sentry.instrumenter=otel
+```
+
+You also have to manually provide the `OpenTelemetryLinkErrorEventProcessor` bean to ensure errors are properly linked to transactions that were created by the OpenTelemetry integration:
```

Comment:
Can remove this section once https://github.com/getsentry/sentry-java/pull/2429 lands.

Reply:
Removed